packages/sidenav/src/components/SideBar/SideBar.styled.js

```diff
@@ -0,0 +1,48 @@
+import styled, { css } from "styled-components";
+import { colors } from "@crave/farmblocks-theme";
+
+import { FULLSCREEN, PUSH, OVERLAY } from "../../constants/variants";
+
+const fullScreenStyle = css`
+  top:0;
+  ${({ expanded }) => expanded && "padding: 16px;"}
+  width: ${({ expanded }) => (expanded ? "100%" : "0px")};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
It's there because of the _Thumb Zone_:
![image](https://user-images.githubusercontent.com/1459283/67336088-1f438c80-f4fb-11e9-9e2f-03645924a56d.png)

Reply:
Great, thanks!